R/readFiles.R

```diff
@@ -283,14 +302,12 @@ read_dat <- function(fname, subsampling = FALSE) {
 #' @param fname The path to the .dat file
 #' @param subsampling Whether to subsample the data or not
 #' @return A list of data.frames
-#' @export
-#'
 read_COCO_dat <- function(fname, subsampling = FALSE) {
   c_read_dat(path.expand(fname), 7, '%')
 }
 
-read_COCO_dat2 <- function(fname, subsampling = FALSE) {
-  select <- seq(5)
+read_COCO_dat2 <- function(fname, DIM, subsampling = FALSE) {
+  select <- seq(5 + DIM)
```

Comment:
Did COCO always store all the x values (all dimensions)? I just forgot this...

Reply:
I think they do, but I still need to test this a bit more. I have turned this off as default now, is that sufficient to prevent issues or should I remove this code for now?

Reply:
It would be better to generate a coco data file using their tool, in high dimensions, say 40D? and check how many columns COCO would put here...

Reply:
It is indeed not always recorded for higher dimensions. I will just remove this code for now